index.html

```diff
@@ -5,8 +5,20 @@
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
         <title>Chiqui RPG</title>
         <link rel="stylesheet" type="text/css" href="styles.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&display=swap" rel="stylesheet">
 </head>
+
+
 <body class="main-color">
+    <div id="container">
+        <h1>Hola soy Emmet-Selch</h1>
+        <span>No me escupas, Esther</span>
+        <input class="searchbar" type="text" id="fname" name="fname" placeholder="Search something: E.G: POTETOS">
```

Comment:
can we indent the attr to make them more readable?

Reply:
DONE